Source/misdat.h

```diff
@@ -25,12 +25,13 @@ enum mienemy_type : uint8_t {
 	TARGET_BOTH,
 };
 
-enum class DamageType : uint8_t {
+enum class FloatingNumberType : uint8_t {

```

Comment:
I think I would prefer that we have way to represent damage type, could you change it so that the page type gets converted to a floating number type rather then then replacing it. That way floating numbers aren't so deeply embeded in the whole code and we can connect other things to the damage type.